target/linux/ramips/dts/mt7621.dtsi

```diff
@@ -488,12 +541,14 @@
 
 	pcie: pcie@1e140000 {
 		compatible = "mediatek,mt7621-pci";
-		reg = <0x1e140000 0x100
-			0x1e142000 0x100>;
-
+		reg = <0x1e140000 0x100     /* host-pci bridge registers */
+			0x1e142000 0x100    /* pcie port 0 RC control registers */
+			0x1e143000 0x100    /* pcie port 1 RC control registers */
+			0x1e144000 0x100>;  /* pcie port 2 RC control registers */
 		#address-cells = <3>;
 		#size-cells = <2>;
 
+		perst-gpio = <&gpio 19 GPIO_ACTIVE_HIGH>;
```

Comment:
Some PCIE changes in "ramips: mt7621: update mt7621.dtsi" are removed in later pcie backport commit. I think it's better to move all pcie node changes into a separated commit instead.